Monika After Story/game/script-topics.rpy

```diff
@@ -1786,12 +1786,20 @@ label monika_rain_holdme:
         m 1dsc "Sorry..."
     return
 
-label monika_holdme_prep:
-    stop music fadeout 1.0
+label monika_holdme_prep(lullaby=True, no_music=True):
 
+    # start the lullaby timer
+    if lullaby and no_music:
+        play music "<silence 1800.0>"
+        queue music "<loop 0.01>mod_assets/sounds/amb/Monika's Lullaby.ogg"
+    # otherwise stop the music without starting the timer
+    elif not lullaby and no_music:
+        stop music
+        
     # clear selected track
     $ songs.current_track = songs.FP_NO_SONG
     $ songs.selected_track = songs.FP_NO_SONG
+    $ persistent.current_track = songs.FP_NO_SONG
```

Comment:
2 things:
1. if we passed in False to no music, then this (+ above 2 lines) will always clear the track, which will have side effects when the music menu is opened again. Clearing all of these should only be done the music changes or is stopped.
2. I don't think it's really necessary or even a good idea to clear the persistent current track.  persistent's current track is set when the user has specifically selected a track. yes we stop the track here, but it doesn't make sense to completey clear the user selection and extend the silence beyond to the session.